docker/pytorch-aarch64/scripts/build-pytorch.sh

```diff
@@ -53,6 +53,28 @@ patch -p1 < $PACKAGE_DIR/onednn.patch
 
 cd $PACKAGE_DIR/$src_repo
 
+if [[ $XLA_BUILD -ne 0 ]]; then
```

Comment:
The Copyright on this file will need to be updated.

Reply:
Nit: I think this can just be `if [[ $XLA_BUILD ]]; then` - consistent with the oneDNN build option on 36.

Does this require a oneDNN build, or are the two entirely decoupled?

Reply:
OneDNN and XLA are entirely decoupled. I will check if i can drop the 0 check.